app/javascript/packs/homePage.jsx

```diff
@@ -93,8 +97,12 @@ if (!document.getElementById('featured-story-marker')) {
         return;
```

Comment:
@Ridhwana since we actually have an intersection observer, we can do away with a lot of this code, and call `    observeDisplayAds()` on load.

A possible refactor would be to modify this functionality so that it can be included instead of it being an undefined global, but in general I think the cleaner thing would be to continue to rely on it.

The key to the existing functionality is that we only call the impression once we scroll to it because of the intersection observer, so I think we want to make sure we retain that functionality and call it instead of having two separate implementations.

You can reference this PR:

https://github.com/forem/forem/pull/19326

Effectively we should be able to put the same call in once the billboards are loaded and it should handle it from there.